localinstall/.env-api

```
@@ -5,4 +5,4 @@ SMTP_HOST=localhost
 SMTP_PORT=25
 EMAIL_SENDER=banana@collabora.com
 EMAIL_PASSWORD=bananapassword
-KERNELCI_API_IMAGE=local/staging-api
+KERNELCI_API_IMAGE=local/staging-kernelci
```

Should it also include the `api` tag?

```suggestion
KERNELCI_API_IMAGE=local/staging-kernelci:api
```

---

It is set by default in Dockerfile:
```
    image: ${KERNELCI_API_IMAGE:-kernelci/staging-kernelci}:${KERNELCI_API_TAG:-api}
```